test/ProperLogger.test.ts

```diff
@@ -0,0 +1,494 @@
+/* eslint-disable no-console, @typescript-eslint/no-empty-function */
```

Comment:
I think the `no-empty-function` disable can be removed now?

Reply:
:thumbsup: `no-console` as well. :thumbsup: